bundles/org.openhab.binding.nuvo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -536,4 +541,10 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="sourceMenu">
```

Comment:
To follow the convention of the existing channel types in this binding, the id should be:
```suggestion
	<channel-type id="source_menu">
```

Or to follow the openHAB naming convention:
```suggestion
	<channel-type id="source-menu">
```

Reply:
Thanks, changed to source_menu for all instances.